tests/sm_int/main.c

```diff
@@ -0,0 +1,11 @@
+// Copyright(C) Microsoft Corporation.All rights reserved.
```

Comment:
Likely MIT as it is in the open

Reply:
Here and everywhere

---
In reply to: [421813759](https://github.com/Azure/azure-c-util/pull/35#discussion_r421813759) [](ancestors = 421813759)